providers/pushwoosh/src/lib/pushwoosh.provider.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  IPushOptions,
+  IPushProvider,
+} from '@novu/stateless';
+
+import { Pushwoosh } from 'web-push-notifications';
+
+type triggeredFunctionType = () => void;
+
+export class PushwooshPushProvider implements IPushProvider {
+  id = 'pushwoosh';
+  channelType = ChannelTypeEnum.PUSH as ChannelTypeEnum.PUSH;
+  pushwooshInstance = new Pushwoosh();
+
+  constructor(
+    private config: {
+      applicationCode: string;
+      defaultNotificationTitle: string;
+      autoSubscribe: boolean;
+      triggeredFunction: triggeredFunctionType;
+      userId?: string;
+      tags?: object;
+      defaultNotificationImageURL?: string;
+    }
```

Comment:
```suggestion
    private config: {
      applicationCode: string;
      defaultNotificationTitle?: string;
      autoSubscribe?: boolean;
      triggeredFunction?: triggeredFunctionType;
      userId?: string;
      tags?: object;
      defaultNotificationImageURL?: string;
    }
```
Based in what I understood from the documents and the API of Pushwoosh.